### DISCUSSION TITLE
Tutorials source code

### DISCUSSION THREAD

**Discussion by realroot2185** [FyroxEngine/Fyrox#620] [Q&A] [answered]
Can I download the tutorials source code?
For example now I'd like to check RPG tutorial code.

**Comment by mrDIMAS** [comment, marked as answer]
Location of the source code was mentioned in the book - https://fyrox-book.github.io/tutorials/tutorials.html (probably not so obvious though). Here's the link - https://github.com/fyrox-book/fyrox-book.github.io/tree/main/src/code/tutorials
